CHANGELOG.md

```diff
@@ -47,6 +47,10 @@ _ `_optional` subpackage for managing optional dependencies
 - `Interval` class now supports degenerate intervals containing only one element
 - `add_fake_results` now directly processes `Target` objects instead of a `Campaign`
 - `path` argument in plotting utility is now optional and defaults to `Path(".")`
+- The default kernel factory now avoids strong jumps by linearly interpolating between
+  two fixed low and high dimensional prior regimes
+- The previous default kernel factory has been renamed to `EDBOKernelFactory` and now
```

Comment:
it was more than just a renaming since in previous attempts fix fix/shift the problem we shifted one of the threshold, that has been reverted and now is fully original logic (hence the phrasing)